### DISCUSSION TITLE
Popping URLs out to a new window

### DISCUSSION THREAD

**Discussion by eskibars** [DustinBrett/daedalOS#373] [Q&A]
Hey, thanks for this great project!  One of the things I wanted to do was to create a bit of a portfolio of some of my work but using daedalOS as a kind of container for it.  I've noticed that many websites have restrictive security policies that don't allow iframing.  e.g. if I wanted to add a link to my GitHub profile inside of daedalOS, I can't do it because GitHub has a frame-blocking security policy in place.  I looked around and I didn't see the answer: is it possible to do something like `_target=new`?  I realize this breaks some of the realism of the OS-in-a-single-browser-window, but I can't think of a great way to avoid something like this without dropping HTML files in that then do the same thing.

**Comment by DustinBrett** [comment]
Hey thanks! I could consider a feature where shortcuts can open in new tabs. But if you wanted to customize it you could also look into making something like that if you desire it to pop up in a new window/tab. There is also isomorphic git which can directly clone repos, so there could be some ideas there for doing a live git connection to your profile. Also you could indeed download the files and host them. I do that for a few apps like JS Paint, it's basically just another app/repo running in a local iframe. Good luck and I will keep this in mind for an additional feature, likely into the shortcut file ini flags.

**Comment by eskibars** [comment]
Thanks!  As a temporary workaround, I did set the URL to be `javascript:window.open("some_url","_blank")&&"Opened in a new window"` which does the trick to pop out a new window, it's just a bit funky that you're left with this open tab you need to close out.  I'd love to have something like a `Target=` attribute in the `.url` files so I could do this a little bit cleaner.  If you'd be up for it, I'm not too much of a typescript person myself, but I suspect I could submit a working pull request to do that in the next couple weeks for your review?

**Reply by DustinBrett** [reply]
I will add the feature to my to do list. Indeed you can submit a PR but I am a bit picky about my project so usually like to do it my own way in the end. But I like the idea and will indeed add it. I think in the shortcut makes sense, but as it is now the open() logic is very much disconnected from window.open concept.
